### ISSUE TITLE
ANCE integration tests

### ISSUE THREAD

**Issue by lintool** [castorini/pyserini#444]
Separating from #382. We need integration tests to prevent future breakage.

**Comment by lintool** [comment]
ANCE tests merged to master in #455 - @MXueguang I'll let you close when ready.

**Comment by MXueguang** [comment]
yeah, I might need to run another round and then close

**Comment by lintool** [comment]
hey @MXueguang I'm getting:

```
======================================================================
FAIL: test_msmarco_doc_ance_bf (test_ance.TestSearchIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch2/jimmylin/pyserini/integrations/test_ance.py", line 73, in test_msmarco_doc_ance_bf
    self.assertAlmostEqual(score, 0.3797, places=4)
AssertionError: 0.3796 != 0.3797 within 4 places

----------------------------------------------------------------------
Ran 50 tests in 101214.386s

FAILED (failures=1)
```

I believe it's fixed by #472, so just FYI.

**Comment by MXueguang** [comment]
Thanks!
